projects/modular_llm/eval_library.py

```diff
@@ -242,13 +214,15 @@ def run_eval(args: ExpertConfig):
         args.router_selector = f"{args.merge_or_route}_router"
 
         selector_config = SelectorConfig.from_training_config(args)
+        if not args.selector_data_id:
+            args.selector_data_id = fetch_prototypes(args, library)
 
         module = MultiExpertModel(
             **vars(train_cfg), selector_config=selector_config
         ).to("cuda")
 
         module.add_experts_from_library(library)
-        patch_prototypes(module, library, args)
+        # patch_prototypes(module, library, args)
```

Comment:
Does `patch_prototypes` still exists?
`mttl/models/expert_model.py` is also referring to it:
```
from projects.modular_llm.eval_library import patch_prototypes
patch_prototypes(self, expert_library, self.selector_config)
```

Reply:
good call